modules/Core/includes/endpoints/ServerInfoEndpoint.php

```diff
@@ -81,6 +81,24 @@ public function execute(Nameless2API $api): void {
             $api->throwError(CoreApiErrors::ERROR_UNABLE_TO_UPDATE_SERVER_INFO, $e->getMessage(), 500);
         }
 
+        // Server query
+        $query_type = Util::getSetting('external_query');
```

Comment:
Does not make sense to keep using `'external_query'` as the setting name now that there are 3 possible values. Could we create a new setting named something like `'server_query_type'`, and use that instead?